pom.xml

```diff
@@ -56,6 +56,25 @@
 			<artifactId>validation-api</artifactId>
 			<version>2.0.1.Final</version>
 		</dependency>
+		<dependency>
+			<groupId>org.flywaydb</groupId>
+			<artifactId>flyway-core</artifactId>
```

Comment:
If you use flyway-mysql this dependency not need because flyway-mysql contains flyway-core.

Reply:
done